UI/src/views/LoginPage.machines.js

```diff
@@ -0,0 +1,119 @@
+/** @format */
+
+import {
+    action,
+    createMachine,
+    immediate,
+    interpret,
+    invoke,
+    guard,
+    reduce,
+    state,
+    transition
+} from "@/robot-vue";
+const registry = require("dijit/registry");
```

Comment:
The LoginPage is Vue and WebElements, yet we're bringing back Dojo as a direct dependency?

Reply:
Yes. I dislike it a lot. Here's the reason: each form has a concept of "being valid", but that checks the standard conditions, not the ones implemented by dojo as part of its widgets. So we can't ask the form whether it's valid or not, because it won't return the correct value.

That's where Ihad to rever to using dojo for now...

Reply:
the registry requirement has now been centralized. Hope that's less upsetting.

Reply:
It only upsets Jest. I'll mock it away.